ci_tools/bot_interaction.py

```diff
@@ -520,7 +528,7 @@ def flagged_as_trusted(pr_id, user):
         else:
             leave_comment(pr_id, message_from_file('bot_commands.txt'))
 
-    elif event['action'] == 'opened':
+    elif event['action'] in ('opened', 'reopened'):
```

Comment:
what does `reopened` do here
is it when you go from `draft` to `open` ?

Reply:
No, it's if you close and then reopen the PR. Sometimes the bot gets stuck when you open it. It's useful to have this as a solution to get the checklist if it didn't trigger properly

Reply:
ah okey I see :+1: